### DISCUSSION TITLE
O que seria mais vantajoso, criar um blog do zero ou usar wordpress?

### DISCUSSION THREAD

**Discussion by renedocket** [frontendbr/forum#1584] [Perguntas e Respostas]
Olá pessoal! 

Estou com um lema de criar meu próprio blog, mas não tenho muita noção de como se faz um totalmente do zero. 

O objetivo com isso é aprender mais sobre desenvolvimento e consequentemente criar conteúdo para compartilhar conhecimentos do que estou estudando. 

Me questionei algumas vezes sobre criar algo com wordpress. O que vocês acham sobre esse assunto e se já fizeram algo parecido?

**Comment by talitaazevedo** [comment]
Bom dia! 
A nível de conhecimento para utilizar como portifólio acho legal, mas eu também acho que tempo é dinheiro então assim para wordpress tem muita coisa já pronta que você só precisa mudar uma coisa ou outra, eu considero muito mais produtivo quando a gente tem pressa pegar alguma coisa pronta e só customizar depois. 

Mas tudo depende  da sua pressa em subir o blog e tals, você poderia por exemplo criar o blog com wordpress, e começar a postar conteúdo  de outras coisas que aprendeu focar em estudos técnicas e tals.

**Comment by vhfmag** [comment]
Tudo depende de seu objetivo! O que você pretende aprender?

Pelo seu post, você pretende usar o blog pra postar conteúdo que vai ser fruto de seus estudos, mas não falou em usar o desenvolvimento do blog em si pra estudar. Nesse caso, Wordpress (e outros CMS, como o Ghost) fazem total sentido e vão te poupar muito tempo.

Mas se você pretende usar o desenvolvimento pra estudar, talvez valha a pena fazê-lo: assim você vai poder estudar CSS (grid 💖, flex, float, variáveis, metodologias, talvez um preprocessador ou um Tailwind da vida...), HTML (acessibilidade, semântica, CEO...) e Javascript (progressive enhancement, service worker, talvez um framework, etc).

Vale dizer também que as opções não são mutualmente exclusivas: você pode criar imediatamente um blog no Wordpress e depois fazer um do zero e portar o conteúdo pra lá 😊

**Comment by wilsonneto-swe** [comment]
Cara se for para aprender, minha opinião:

- Manda bala com wordpress que em uma manhã você coloca um site muito bonito online

- Use seu próprio comportamento no wordpress para servir de guia para criar um desenvolvido por você do zero.

- Coloca a mão na massa e faça o seu, com bem menos features que o wordpress, mas se for para aparender faça o seu tbm.

- Eu fiz vários blogs do zero, até para cliente, a maioria em PHP, mas alguns até usando .Net, em todos os casos fiz do zero pq queria exercitar meu algoritimo e pegar o jeito com as linguagem, mas se um cliente hoje me pede apenas um blog para artigos e vejo que o wordpress atende, em uma manhã eu configuro e tenho o treinamento de como usar gravado, assim consigo fazer o projeto em um dia... Uso o tempo maior para focar em desenvolvimento de aplicações e apps, que dão mais retorno que blogs ou sites institucionais.

Minha opinão, forte abraço!

**Comment by felipefialho** [comment]
Fala @renedocket, beleza?

Eu daria um passo antes, penso que nesse momento o processo de desenvolver seu blog e entender todas as etapas vai trazer ganhos enormes na sua evolução profissional.

Você vai aprender um monte de coisa, poucas coisas são melhores para aprendizado do que trabalhar no seu próprio/site blog, e inicialmente, penso que isso é mais importante do que a geração do conteúdo, que será uma consequência disso.

Nesse [artigo conto como foi desenvolver a última versão do meu blog](https://www.felipefialho.com/blog/como-foi-desenvolver-meu-novo-blog-usando-o-gatsbyjs/), talvez te ajude.
